src/shared/components/simpleTable/SimpleTable.spec.tsx

```diff
@@ -1,35 +1,47 @@
 import React from 'react';
 import { assert } from 'chai';
-import { shallow, mount } from 'enzyme';
+import { shallow } from 'enzyme';
 import sinon from 'sinon';
 import SimpleTable from "./SimpleTable";
 
 describe('SimpleTable', () => {
 
     let table:any;
 
-    before(()=>{
+    const headers:any = [<th>one</th>];
+    const rows:any = [<tr><td>data</td></tr>, <tr><td>data</td></tr>];
+    const classNameTrue:string = 'test-class';
+    const classNameFalse:undefined = undefined;
 
-        let headers:any = [<th>one</th>];
-        let rows:any = [<tr><td>data</td></tr>, <tr><td>data</td></tr>];
+    before(()=> {
 
         table = shallow(<SimpleTable headers={headers} rows={rows} />);
```

Comment:
you don't need to make any change, but i think we have typings for enzyme, so one would then get autocomplete on the enzyme methods (e.g. find).